src/main/java/fr/insalyon/pldagile/view/Window.java

```diff
@@ -299,11 +306,30 @@ public void orderListRequests(ArrayList<Pair<Long, String>> steps, Map<Long, Req
             {
                 item = new RequestItem("Pickup at " + requests.get(step.getKey()).getPickup().getIntersection().getId(), "Duration: " + requests.get(step.getKey()).getPickup().getDuration(), step.getKey(), "Pickup",index);
                 items.add(item);
+
+                MapPoint mapPoint = new MapPoint(requests.get(step.getKey()).getPickup().getIntersection().getCoordinates().getLatitude(), requests.get(step.getKey()).getPickup().getIntersection().getCoordinates().getLongitude());
```

Comment:
pls spilt this line up its super long and hard to read

Reply:
Done